packages/mui-base/src/SelectUnstyled/useSelect.ts

```diff
@@ -89,7 +88,7 @@ function useSelect<TValue>(props: UseSelectParameters<TValue>) {
     focusListboxIfRequested();
   };
 
-  const handleListboxRef = useForkRef(intermediaryListboxRef, updateListboxRef);
+  const handleListboxRef = useForkRef(useForkRef(listboxRefProp, listboxRef), updateListboxRef);
```

Comment:
Just for my learning, why did you decide not to store the ref in `intermediaryListboxRef`?

Reply:
I thought that not introducing a variable here could help in readability (you don't have to think if intermediary ref is used anywhere else besides being passed into another useForkRef).

Reply:
Ah, I see. Thanks.